src/commands/logout/index.ts

```diff
@@ -0,0 +1,16 @@
+import fs from 'fs'
+import { togglebotSleep } from '../../ui/togglebot'
+import Base from '../base'
+
+export default class Logout extends Base {
+    static hidden = false
+    static description = 'Log in through the DevCycle Universal Login. This will open a browser window'
+    static examples = []
+
+    public async run(): Promise<void> {
+        if (fs.existsSync(this.authPath)) {
+            fs.rmSync(this.authPath)
+        }
+        console.log(togglebotSleep) 
```

Comment:
do we not want all logs to go through the built-in logging system? I imagine part of the reason for doing that is that it would let us properly implement a "--silent" flag or change the amount of logging output

Reply:
When we do that we should use our own logging function. The oclif solution requires any code that wants to write to the log to have a reference to the Command, which is probably my biggest gripe about oclif.